### PR TITLE
Fixes for ARM64

### PR DESCRIPTION
**Pull request by bpulliam**
## Description
- Remove configurations for arm (32 bit, not 64 bit)
- Add ARM64 configurations to the solutions where it was missing
- Fix mis-configured ARM64 configurations, where it used Any CPU or Win32 instead of ARM64
- Fix issues where the platform is defined as arm64 instead of ARM64.  This can cause issues in multithreaded builds.
- Fixed EnablePreviewMsixTooling to be EnableMsixTooling
- Fixed quotes in file headers

## Target Release
WinAppSDK 1.4

## Checklist

- [ ] Samples build and run using the Visual Studio versions listed in the [Windows development docs](https://docs.microsoft.com/windows/apps/windows-app-sdk/set-up-your-development-environment?tabs=stable#2-install-visual-studio).
- [X] Samples build and run on all supported platforms (x64, x86, ARM64) and configurations (Debug, Release).
- [ ] Samples set the minimum supported OS version to Windows 10 version 1809.
- [X] Samples build clean with no warnings or errors.
- [ ] **[For new samples]**: Samples have completed the [sample guidelines checklist](https://github.com/microsoft/WindowsAppSDK-Samples/blob/main/docs/samples-guidelines.md#checklist) and follow [standardization/naming guidelines](https://github.com/microsoft/WindowsAppSDK-Samples/blob/main/docs/samples-guidelines.md#standardization-and-naming).
- [ ] If I am onboarding a new feature, then I must have correctly setup a new CI pipeline for my feature with the correct triggers and path filters laid out in the "Onboarding Samples CI Pipeline for new feature" section in samples-guidelines.md.
- [ ] I have commented on my PR `/azp run SamplesCI-<FeatureName>` to have the CI build run on my branch for each of my FeatureName my PR is modifying. This must be done on the latest commit on the PR before merging to ensure the build is up to date and accurate. Warning: the PR will not block automatically if this is not run due to '/azp run' limitation on triggering more than 10 pipelines.

### CODE REVIEW

File: Samples/AppLifecycle/RestartRegistration/cs-winui-packaged/cs-winui-packaged.csproj [hunk 1]
```diff
@@ -5,7 +5,7 @@
     <TargetPlatformMinVersion>10.0.17763.0</TargetPlatformMinVersion>
     <RootNamespace>cs_winui_packaged</RootNamespace>
     <ApplicationManifest>app.manifest</ApplicationManifest>
-    <Platforms>x86;x64;arm64</Platforms>
+    <Platforms>x86;x64;ARM64</Platforms>
     <RuntimeIdentifiers>win10-x86;win10-x64;win10-arm64</RuntimeIdentifiers>
     <UseWinUI>true</UseWinUI>
     <EnablePreviewMsixTooling>true</EnablePreviewMsixTooling>
```
Comment (Scottj1s) [inline Samples/AppLifecycle/RestartRegistration/cs-winui-packaged/cs-winui-packaged.csproj:11]: EnablePreviewMsixTooling now obsoleted by EnableMsixTooling

File: Samples/AppLifecycle/StateNotifications/cs-wpf-packaged/CsWpfState/CsWpfState.csproj [hunk 2]
```diff
@@ -4,8 +4,8 @@
     <OutputType>WinExe</OutputType>
     <TargetFramework>net6.0-windows10.0.19041.0</TargetFramework>
     <UseWPF>true</UseWPF>
-    <Platforms>x64;x86</Platforms>
-    <RuntimeIdentifiers>win10-x64;win10-x86;win-x86;win-x64</RuntimeIdentifiers>
+    <Platforms>x64;x86;ARM64</Platforms>
+    <RuntimeIdentifiers>win10-x64;win10-x86;win10-arm64;win-x86;win-x64;win-arm64</RuntimeIdentifiers>
```
Comment (Scottj1s) [inline Samples/AppLifecycle/StateNotifications/cs-wpf-packaged/CsWpfState/CsWpfState.csproj:8]: win-* RIDs imply execution on Win7, which WinAppSDK doesn't support - should remove throughout all samples

File: Samples/ResourceManagement/cs-wpf/wpf_packaged_app/wpf_packaged_app.csproj [hunk 3]
```diff
@@ -4,7 +4,7 @@
     <OutputType>WinExe</OutputType>
     <TargetFramework>net6.0-windows10.0.19041.0</TargetFramework>
     <TargetPlatformMinVersion>10.0.17763.0</TargetPlatformMinVersion>
-    <Platforms>x86;x64;arm64</Platforms>
+    <Platforms>x86;x64;ARM64</Platforms>
     <RuntimeIdentifiers>win-x86;win-x64</RuntimeIdentifiers>
```
Comment (Scottj1s) [inline Samples/ResourceManagement/cs-wpf/wpf_packaged_app/wpf_packaged_app.csproj:8]: more RID checks

**Comment by Scottj1s** [comment]
>   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|arm64'">

nit: ARM64 (upper case)

---
Refers to: Samples/AppLifecycle/Instancing/cs-winui-packaged/CsWinUiDesktopInstancing/CsWinUiDesktopInstancing/CsWinUiDesktopInstancing.csproj:24 in b9042e1. [](commit_id = b9042e1946adafebb250a135ef5ecd6d99e61c5a, deletion_comment = False)

**Review by Scottj1s — APPROVED** [review]
approved with feedback

File: Samples/AppLifecycle/RestartRegistration/cs-winui-packaged/cs-winui-packaged.csproj (hunk 1 above)
Comment (bpulliam) [reply Samples/AppLifecycle/RestartRegistration/cs-winui-packaged/cs-winui-packaged.csproj:11]: fixed.

File: Samples/AppLifecycle/StateNotifications/cs-wpf-packaged/CsWpfState/CsWpfState.csproj (hunk 2 above)
Comment (bpulliam) [reply Samples/AppLifecycle/StateNotifications/cs-wpf-packaged/CsWpfState/CsWpfState.csproj:8]: I'll do this separately

**Pull request merged**